Library/include/CSP/Multiplayer/SpaceEntitySystem.h

```diff
@@ -264,6 +264,11 @@ class CSP_API SpaceEntitySystem
 	/// It is highly advised not to call this function unless you know what you are doing.
 	void RetrieveAllEntities();
 
+	/// @brief Destroys the client's local view of all currently known entities.
+	///
+	/// They still reside on the server, however they will not be accessible in the client application.
+	void LocalDestroyAllEntities();
```

Comment:
Does a client app need to call this manually ?

Reply:
@lyubomirlichev it is called below in the SpaceSystem.

Reply:
That's right, a client does _not_ need to call this, but SpaceSystem needs access to it in order to invoke it on exiting a space.

I elected to make it public as it's the symmetric opposite of `RetreiveAllEntities` just above, which is also public; it felt neater than adding a new `friend` qualifier.

Reply:
Thank you, sounds good 👍